dataformats/multigrid/inc/RunSpecTest.cpp

```diff
@@ -0,0 +1,31 @@
+/** Copyright (C) 2016 European Spallation Source ERIC */
+
+#include <RunSpec.h>
+#include <test/TestBase.h>
+
+
+
+class RunSpecTest : public TestBase {
+protected:
+};
+
+/** Test cases below */
+
+TEST_F(RunSpecTest, DefaultConstructor) {
+  static const int start{1};
+  static const int end{999};
```

Comment:
Seeing as each test is only run once, I do not really see a reason for these variables to be static. On the other hand, the static keyword is not a problem as such.